src/App.test.jsx

```diff
@@ -20,12 +20,22 @@ const user = {
 }
 
 // 🚨 Create your server
+const server = setupServer(
+  rest.get('https://uzgiamkrbapxufnwdrja.supabase.co/rest/v1/users', (req, res, ctx) => {
+    const select = req.url.searchParams.get('select')
+    if (select === '*') {
+      return res(ctx.json([user]))
+    }
+
+    return res(ctx.json())
```

Comment:
We should send an error instead of `ctx.json()` in the case your application not making a request with the proper query params
https://mswjs.io/docs/recipes/request-assertions